tests/protocol/mockwiredata.py

```diff
@@ -180,6 +184,7 @@ def reload(self):
         self.ga_manifest = load_data(self.data_files.get("ga_manifest"))
         self.trans_prv = load_data(self.data_files.get("trans_prv"))
         self.trans_cert = load_data(self.data_files.get("trans_cert"))
+        self.imds_info = json.loads(load_data(self.data_files.get("imds_info")))
```

Comment:
There's an added dependency on the `json` module here because the json files (i.e. `valid.json` and `unicode.json`) include data returned by both `compute` and `network`.